### PR TITLE
mysqld: add page

### LINKED ISSUE #1177: page request: mysqld
This is the command that runs the server

https://linux.die.net/man/1/mysqld

### PR DESCRIPTION
**Pull request by marchersimon**
This PR is a continuation of #1438, since it doesn't seem to be very active anymore. This also closes #1177.

### CODE REVIEW

File: pages/common/mysqld.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mysqld
+
+> Start the MariaDB database server.
+> More information: <https://dev.mysql.com/doc/refman/8.0/en/mysqld.html>.
```
Comment (bl-ue) [inline pages/common/mysqld.md:4]: We could remove the version...
```suggestion
> More information: <https://dev.mysql.com/doc/refman/en/mysqld.html>.
```

File: pages/common/mysqld.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Start the MariaDB database server:
+
+`mysqld`
+
+- Display error messages:
```
Comment (bl-ue) [inline pages/common/mysqld.md:10]: This option is [Windows only](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_console), so:
```suggestion
- Start the database server, printing error messages to the console (Windows only):
```

File: pages/common/mysqld.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysqld --console`
+
+- Specify the log file location:
```
Comment (bl-ue) [inline pages/common/mysqld.md:14]:
```suggestion
- Start the server, saving logging output to a custom log file:
```

File: pages/common/mysqld.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysqld --log={{path/to/file}}`
```
Comment (bl-ue) [inline pages/common/mysqld.md:16]:
```suggestion
`mysqld --log={{path/to/file.log}}`
```

File: pages/common/mysqld.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Print the default arguments and exit:
```
Comment (bl-ue) [inline pages/common/mysqld.md:18]:
```suggestion
- Print the default arguments and their values and exit:
```

File: pages/common/mysqld.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysqld --print-defaults`
+
+- Use default arguments from a specified file:
```
Comment (bl-ue) [inline pages/common/mysqld.md:22]:
```suggestion
- Start the server, reading arguments and values from a file:
```

File: pages/common/mysqld.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysqld --defaults-file={{path/to/file}}`
+
+- Specify the port number to use for connections:
```
Comment (bl-ue) [inline pages/common/mysqld.md:26]:
```suggestion
- Start the server and listen on a custom port:
```

File: pages/common/mysqld.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysqld --port={{port}}`
+
+- Show all help options:
```
Comment (bl-ue) [inline pages/common/mysqld.md:30]:
```suggestion
- Show all help options and exit:
```

File: pages/common/mysqld.md (hunk 1 above)
Comment (bl-ue) [inline pages/common/mysqld.md:4]: This could be shortened.
```suggestion
> More information: <https://dev.mysql.com/doc/refman/en/mysqld.html>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
I think longer descriptions are better, because they're a bit more explicit.

File: pages/common/mysqld.md (hunk 2 above)
Comment (marchersimon) [reply pages/common/mysqld.md:10]: `Write error output on screen; don't remove the console window on windows.`

I think that means that on linux and macos, errors get printed in the console and in windows the console deosn't get closed as it normally would. But maybe I'm misinterpreting it.

**Comment by marchersimon** [comment]
@bl-ue they definetely are more clear and readable.

File: pages/common/mysqld.md (hunk 2 above)
Comment (bl-ue) [reply pages/common/mysqld.md:10]: Where'd you get that? I can't find it :/

Comment (marchersimon) [reply pages/common/mysqld.md:10]: That's somewhere in the 3000 lines of the help page, when you run `mysqld --verbose --help` :sweat_smile:. 
image

Comment (bl-ue) [reply pages/common/mysqld.md:10]: Okay then, not sure what the docs are talking about 🤷🏻 suggestion updated.

File: pages/common/mysqld.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# mysqld
+
+> Start the MariaDB database server.
```
Comment (MasterOdin) [inline pages/common/mysqld.md:3]: It's funny to have this refer to `MariaDB` in help text, but link to mysql docs. Probably better to just say `MySQL` and people using `MariaDB` should know that they're largely interchangeable.

**Review by bl-ue — APPROVED** [review]

**Review by MasterOdin — APPROVED** [review]

**Pull request merged**